Index: helloworld.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"\" Prueba de git\"\"\"\r\nprint(\"bienvenido al mundo del Git\")\r\nprint(\"che forzapaite\")\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/helloworld.py b/helloworld.py
--- a/helloworld.py	
+++ b/helloworld.py	
@@ -1,5 +1,4 @@
 """" Prueba de git"""
 print("bienvenido al mundo del Git")
 print("che forzapaite")
-
-
+print("jaikua la upeaa")
